### ISSUE TITLE
Add ontology links to a prominent location on the spec home page

### ISSUE THREAD

**Issue by bact** [spdx/spdx-spec#1116]
Should we have links to ontology at a prominent location (top or bottom) on the spec home page?

- Will be handy, easy to find
- Announce/remind official/preferred URLs 
  - For example, http://spdx.org/rdf/3.0.0/spdx-model.ttl instead of https://spdx.github.io/spdx-spec/v3.0.0/rdf/spdx-model.ttl
- Similar to what OMG has on its website at the bottom of the spec home page

![OMG](https://github.com/user-attachments/assets/ab4f7445-c172-4b85-8a2e-61648fac73c3)
https://www.omg.org/spec/Commons/1.1

**Comment by zvr** [comment]
Yes, we should, but we don't even publish the ontology yet, @bact.

The simple `.ttl` that we generate will be further processed and generate a whole website, like https://spdx.org/rdf/terms/ 

After the deadlines of next week ;-)
